### DISCUSSION TITLE
Academic conference 2023 orientation, 2022-12-30 Fri 23:00

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#237] [General]
Academic conference 2023 오리엔테이션이 
12월 30일 금요일 오후 11시에 진행됩니다.

작년과 마찬가지로 게더타운에서 진행합니다.
https://app.gather.town/app/usIEdVq3puyhhTQK/Academic%20conference%202023

진행 순서는 다음과 같습니다.
- 각자 자기 소개
- Academic conference 2023 진행 방식 소개
- 책 읽는 순서 결정
- 워크샵 여행지 소개 및 후보지 논의

입니다.

내년에도 재밌고 유익하게 진행했으면 좋겠습니다.

**Comment by TaeHyoungKwon** [comment]
오.. 이번에 새로 추가된 멤버도 많으신가요!? ㅎㅎ 😄

**Reply by jongfeel** [reply]
예상에는 한 분 혹은 두분 더 오실 것 같고
기존 멤버 분들까지 하면 6명 7명 될 것 같습니다.

**Reply by TaeHyoungKwon** [reply]
오 글쿤요 👍🏻 기대됩니다!

**Reply by jongfeel** [reply]
제가 메일도 보내 드렸었는데 회신 한번 주시면 좋겠습니다.
